### DISCUSSION TITLE
Implement "next highlight" feature for replays

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#26779] [Q&A] [answered]
This feature comes from Dota 2 and allows user to jump to a highlight in the game. 

In osu! game highlights are:

- Stream sections
- User's misses
- Jump sections

![20240129201642_1](https://github.com/ppy/osu/assets/86934170/92b55f93-d8b1-4377-86ce-49d7ebb9066f)

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/5681.
